salt/utils/event.py

```diff
@@ -529,8 +529,8 @@ def _get_event(self, wait, tag, match_func=None, no_block=False):
                 ret = {'data': data, 'tag': mtag}
             except KeyboardInterrupt:
                 return {'tag': 'salt/event/exit', 'data': {}}
-            except (tornado.iostream.StreamClosedError, RuntimeError):
-                return None
+            except (tornado.iostream.StreamClosedError, RuntimeError) as exc:
+                raise exc
```

Comment:
Since all you are doing is catching an exception and then rethrowing, omitting this code entirely would do the same thing (since all other exceptions aren't caught here) and be cleaner.